src/hundred_pushups/core.cljc

```diff
@@ -180,10 +180,29 @@
   (time/today-at (time/hour (time.format/parse day-of-formatter time-string)) 00))
 
 (defn day-symbol [day-num]
-  (keyword  (["monday" "tuesday" "wednesday" "thursday" "friday" "saturday" "sunday"] day-num)))
+  (keyword  (["monday" "tuesday" "wednesday" "thursday" "friday" "saturday" "sunday"] (- day-num 1))))
```

Comment:
Minor thing, just FYI: you can use `inc` instead of `(+ x 1)` and `dec` instead of `(- x 1)`. Can come in handy since it's a pretty common operation.